scripts/boundary.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+import geopandas as gpd
+from shapely.ops import unary_union
+
+# Read shapefile of all 300+ AOIs.
+gdf = gpd.read_file("input/AOI_v2_1.shp")
+
+# Get the boundary (perimeter) of entire set of AOIs.
+gdf = gpd.GeoDataFrame(
+    gpd.GeoSeries(unary_union(gdf.geometry.values)), columns=["geometry"]
```

Comment:
This is great, and, I think we should merge this now but we need to update it after the polygon simplification is done (#73)